packages/kbn-alerts-as-data-utils/src/schemas/generated/stack_schema.ts

```diff
@@ -70,6 +70,7 @@ const StackAlertRequired = rt.type({
 });
 const StackAlertOptional = rt.partial({
   'kibana.alert.evaluation.conditions': schemaString,
+  'kibana.alert.evaluation.threshold': schemaStringOrNumber,
```

Comment:
Why is this `StringOrNumber`? In the mapping I saw:

```
      [ALERT_EVALUATION_THRESHOLD]: {
        type: 'scaled_float',
        scaling_factor: 100,
        required: false,
      },
```

Is including string in the type for future use cases?

Reply:
The schema is generated this way because Elasticsearch numerical mappings by default will coerce a string version of the number so you could set a value to `100` or `'100'`. This is determined in the schema generation script though so if we want to be more strict with the typings, we can.